src/ApiPayloadConverter.js

```diff
@@ -92,6 +92,62 @@ class ApiPayloadConverter {
 
     return payload;
   }
+
+  /**
+   * Converts the mail payload for the ZeptoMail.
+   * @param {Object} originalPayload The original payload
+   * @param {String} originalPayload.api The provider API version.
+   * @param {Object} originalPayload.payload The payload to convert to be compatible with the provider API.
+   * @returns {Object} The payload according to ZeptoMail SDK specification.
+   */
+  static zeptomail(originalPayload) {
```

Comment:
```suggestion
  static zeptomail(data) {
```

Just to avoid confusion, I would call the container object `data` or something else instead of `originalPayload`, since it already has a child `payload`.